src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -26,4 +26,21 @@
     <PackageReference Include="System.Threading.ThreadPool" Version="4.4.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\FrameFeatureCollection.cs" />
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\KnownHeaders.cs" />
+    <CodeGenOutput Include="Internal\Http\FrameHeaders.Generated.cs" />
+    <CodeGenOutput Include="Internal\Http\Frame.Generated.cs" />
+  </ItemGroup>
+
+  <Target Name="GenerateCode"
+          Inputs="@(CodeGenInput)"
+          Outputs="@(CodeGenOutput)"
+          BeforeTargets="CoreCompile">
+    <Exec Command="dotnet restore"
```

Comment:
We shouldn't need it. It should be picked up as a part of the normal restore operations.